weareinreach/inreach-catalog
# Feature/1199130629299231/add fave

## Description



Asana tickets: 
 - https://app.asana.com/0/1132189118126148/1199130629299231/f
 - https://app.asana.com/0/1132189118126148/1200049576032404/f

## PR Checklist




## How to Test

Go to catalog homepage
Sign In/Up
Select a country from the country dropdown
Type in a location and click "search" to search catalog
Click on red heart button (to the right of any organization profile page on the search results page
If no list exists, default list "my list" will get created and item added to it
if a list exists, user will be able to choose which list the item should be added to

Expected Behaviour:
Organization is added to user's selected"favourites" list

Actual Behaviour:
Organization is not being added to user's "favourites" list
